src/server/app.js

```diff
@@ -98,7 +98,31 @@ export default async () => {
 
   app.use(i18n({
     translationsPath: path.join(__dirname, 'i18n'),
-    siteLangs: ['en', 'fr', 'de', 'cy', 'es', 'pl'],
+    siteLangs: [
+      'en',
+      'fr',
+      'de',
+      'cy',
+      'es',
+      'pl',
+      'bg',
+      'hr',
+      'cz',
+      'nl',
+      'ee',
+      'gr',
+      'hu',
+      'it',
+      'lv',
+      'lt',
+      'pt',
+      'ro',
+      'ru',
+      'rs',
```

Comment:
Is this meant to be Serbian? I think it should be 'sr' if so.

Reply:
The 2 character languages codes are based on the list here: https://www.iban.com/country-codes. SR would be used for Suriname